### PR TITLE
gitlab-runner: add page

### PR DESCRIPTION
**Pull request by prudloff-insite**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/gitlab-runner.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# gitlab-runner
+
+> Run GitLab CI jobs.
+> More information: <https://docs.gitlab.com/runner/>.
+
+- Register a runner:
+
+`sudo gitlab-runner register --url {{https://gitlab.example.com/}} --registration-token {{token}} --name {{runner_name}}`
```
Comment (navarroaxel) [inline pages/common/gitlab-runner.md:8]: Based on https://docs.gitlab.com/runner/register/index.html the trailing `/` is not required.
Also if I wonder if `runner_name` is not redundant. Could we use just `{{name}}`? or just `runner_name` for the first example:
```suggestion
`sudo gitlab-runner register --url {{https://gitlab.example.com}} --registration-token {{token}} --name {{name}}`
```

File: pages/common/gitlab-runner.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Register a runner with a specific executor:
+
+`sudo gitlab-runner register --url {{https://gitlab.example.com/}} --registration-token {{token}} --name {{runner_name}} --executor {{docker}}`
```
Comment (navarroaxel) [inline pages/common/gitlab-runner.md:12]:
```suggestion
`sudo gitlab-runner register --url {{https://gitlab.example.com}} --registration-token {{token}} --name {{name}} --executor {{docker}}`
```

File: pages/common/gitlab-runner.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Unregister a runner:
+
+`sudo gitlab-runner unregister --name {{runner_name}}`
```
Comment (navarroaxel) [inline pages/common/gitlab-runner.md:16]:
```suggestion
`sudo gitlab-runner unregister --name {{name}}`
```

File: pages/common/gitlab-runner.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Get the status of the runner service:
```
Comment (navarroaxel) [inline pages/common/gitlab-runner.md:18]:
```suggestion
- Display the status of the runner service:
```

File: pages/common/gitlab-runner.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+`sudo gitlab-runner status`
+
+- Restart the runner service:
+
+`sudo gitlab-runner restart`
+
+- Verify registered runners are running:
```
Comment (navarroaxel) [inline pages/common/gitlab-runner.md:26]: The doc says this is the output:
```
Verifying runner... is alive                        runner=fee9938e
```
Do the command say if the runner is just connected or if running?

File: pages/common/gitlab-runner.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
+# gitlab-runner
+
+> Run GitLab CI jobs.
+> More information: <https://docs.gitlab.com/runner/>.
+
+- Register a runner:
+
+`sudo gitlab-runner register --url {{https://gitlab.example.com/}} --registration-token {{token}} --name {{runner_name}}`
+
+- Register a runner with a specific executor:
```
Comment (navarroaxel) [inline pages/common/gitlab-runner.md:10]: You wrote a example for docker. Maybe this is better:
```suggestion
- Register a runner with a docker executor:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hey @prudloff-insite. Thanks, I left some comments.

File: pages/common/gitlab-runner.md (hunk 5 above)
Comment (prudloff-insite) [reply pages/common/gitlab-runner.md]: I changed it to use the same wording as the doc.

**Review by navarroaxel — APPROVED** [review]
It's hard to review your PR, because the docs is under DDoS https://twitter.com/gitlabstatus/status/1318136465231028224 :rofl:

File: pages/common/gitlab-runner.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
+# gitlab-runner
+
+> Run GitLab CI jobs.
```
Comment (navarroaxel) [inline pages/common/gitlab-runner.md:3]: You're describing what a runner does, not the command itself. This sounds better for me:
```suggestion
> A CLI tool for managing your GitLab runners.
```
What do you think?

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Oh, I forgot this.

**Review by navarroaxel — APPROVED** [review]

File: pages/common/gitlab-runner.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+# gitlab-runner
+
+> A CLI tool for managing your GitLab runners.
```
Comment (sbrl) [inline pages/common/gitlab-runner.md:3]: Perhaps this will read better?
```suggestion
> CLI tool for managing GitLab runners.
```

Comment (prudloff-insite) [reply pages/common/gitlab-runner.md:3]: See https://github.com/tldr-pages/tldr/pull/4758#discussion_r508295350.
Is there a consensus for this kind of things?

**Review by sbrl — APPROVED** [review]
Thanks, @prudloff-insite!

**Pull request merged**